Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitRequest.swift

```diff
@@ -33,7 +33,6 @@ import BraintreeCore
     ///   - mandateType: Optional. The `BTSEPADebitMandateType`. If not set, defaults to `.oneOff`
     ///   - billingAddress: Required. The user's billing address.
```

Comment:
For the params documented as "Required", should those params be non-optional?

Reply:
So this brings into question how we want to initialize objects in the SDK (and it is a little goofy). These are all optional and defaulted to `nil` so that we can use the dot syntax like:
```
var sepaDirectDebitRequest = BTSEPADirectDebitRequest()
sepaDirectDebitRequest.accountHolderName = "John Doe"
sepaDirectDebitRequest.iban = "FR891751244434203564412313"
...
```

If we make these required we would _always_ require merchants to initialize the required parameters like:
```
var sepaDirectDebitRequest = BTSEPADirectDebitRequest(
    accountHolderName: "John Doe", 
    iban: "FR891751244434203564412313", 
    customerID: "1234", 
    billingAddress: <a_long_billing_address_that_i_dont_want_to_type>
)
```

Then you could still use the dot syntax after initializing the required params (or add them to the init). We _could_ do that but it breaks convention of what the rest of the SDK does, so seems like something we would want to do at a later point when the entire SDK is in Swift so that we are consistent across the SDK. Thoughts?

Reply:
While we were doing this work initially I had the same question! I opened a GitHub discussion to see if merchants had any options but no one responded 😭 : https://github.com/braintree/braintree_ios/discussions/800

Reply:
> We could do that but it breaks convention of what the rest of the SDK does, so seems like something we would want to do at a later point when the entire SDK is in Swift so that we are consistent across the SDK.

Gotcha, thank you for the explanation. I agree, we should 🏈 on it when we're ready to be consistent across all modules.

Reply:
We could throw and validate / throw on error when the request is made. We could use empty strings, etc. as default values to allow the types to be non-optional.